docs/source/manual/configuration.rst

```diff
@@ -105,13 +105,15 @@ Request Log
 .. code-block:: yaml
 
     server:
-      requestLog: 
-        timeZone: UTC
+      requestLog:
+        - type: SLF4J
```

Comment:
Will users have to add this `type` reference to all of their existing configuration files if this gets merged in?  Is there anyway to default the value to `SLF4J` to ease upgrading?  If we can't support that, we need to ensure we update all of the various configuration files spread throughout the `dropwizard-example` and documentation (like the getting started pages).